man/systemd-boot.xml

```diff
@@ -73,8 +73,8 @@
       <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry> for
       details.</para></listitem>
 
-      <listitem><para>An EFI variable set by the boot loader informs the OS about the ESP partition used
-      during boot. This is then used to automatically mount the correct ESP partition to
+      <listitem><para>An EFI variable set by the boot loader informs the OS about the EFI system partition used
+      during boot. This is then used to automatically mount the correct EFI system partition to
```

Comment:
Please make this "EFI System Partition" to match BOOT_LOADER_SPECIFICATION.

Reply:
As you wish. Though, personally I prefer to use sentence case like it's done in [the Wikipedia article](https://en.wikipedia.org/wiki/EFI_system_partition).